api/types/okta.go

```diff
@@ -291,3 +300,23 @@ func (o *OktaAssignmentActionV1) GetTargetType() string {
 func (o *OktaAssignmentActionV1) GetID() string {
 	return o.Target.Id
 }
+
+// Assignments is a list of Okta assignment resources.
+type OktaAssignments []OktaAssignment
+
+// AsResources returns these Okta assignments as resources with labels.
+func (o OktaAssignments) AsResources() (resources ResourcesWithLabels) {
+	for _, assignment := range o {
```

Comment:
When an output slice is a 1:1 mapping from an input slice, consider well pre-allocating the correct size:

    resources = make(ResourcesWithLabels, 0, len(o))